cds/modules/records/tasks.py

```diff
@@ -223,3 +228,108 @@ def file_integrity_report():
         recipients = [current_app.config['CDS_ADMIN_EMAIL']]
         msg = Message(subject, sender=sender, recipients=recipients, body=body)
         current_app.extensions['mail'].send(msg)
+
+
+def format_subformats_integrity_report(report):
+    """Format the email body for the file integrity report."""
+    lines = []
+
+    for entry in report:
+        lines.append('Message: {}'.format(str(entry.get('message'))))
+        lines.append('Record: {}'.format(entry.get('recid')))
+        lines.append('Report number: {}'.format(entry.get('report_number')))
+        lines.append('Other: {}'.format(entry.get('other')))
```

Comment:
What is `other`?
It looks like bucket/version/key, but can you please format it better? For example:
```
Record bucket:
Object Version id:
Filename: 
```